package.json

```diff
@@ -40,15 +41,13 @@
     "gulp-util": "^3.0.7",
     "is-json": "^2.0.1",
     "jasmine-core": "~2.4.1",
-    "karma": "^1.3.0",
-    "karma-coverage": "^1.0.0",
-    "karma-jasmine": "^1.0.2",
-    "karma-jspm": "^2.1.0",
-    "karma-junit-reporter": "^1.0.0",
-    "karma-phantomjs-launcher": "^1.0.0",
+    "karma": "^1.5.0",
```

Comment:
I had to explicitly install karma 1.5.0, because npm<5
Tested. Works.

Reply:
Changed to `"karma": "~1.5.0",`